### PR TITLE
LPS-168119 Add Autom. Test FDSCustomView#HasOptionToEditName

### PR DESCRIPTION
**Pull request by eltonccdantas**
**Test Plan**

- Given FDS Sample Portlet
- And Given Custom tab
- And Given created custom view
- When open ellisis next to custom view
- Then actions to edit custom view name is available

**Jira Ticket:**
https://issues.liferay.com/browse/LPS-168119

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
Just started reviewing :)

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase [hunk 1]
```diff
@@ -191,6 +191,41 @@ definition {
...
+				locator1 = "FrontendDataSet#SELECT_OPTION_CUSTOM_VIEW");
+
+			FDSCustomView.createNewView(key_nameView = "TEST");
+
+			Click(locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
+
+			Click(
+				key_itemName = "Save View",
+				locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
+		}
```
Comment (john-co) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:216]: tbh, I think we only need one line: FDSCustomView.createNewView(key_nameView = "TEST");

all other lines here are not needed.

**Review by john-co — APPROVED** [review]
Please make suggested change. Besides that looks good to me!

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase (hunk 1 above)
Comment (eltonccdantas) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:216]: Ok,

**Comment by john-co** [comment]
ci:forward

**Comment by john-co** [comment]
Looks good to me! :sunglasses:

**Pull request closed without merging**